stories/index.js

```diff
@@ -0,0 +1,10 @@
+// Add global styles
+import 'amo/components/App/styles.scss';
+import 'core/css/inc/lib.scss';
+
+// These styles override some App styles that make it hard to see
+// in storybook.
+import './setup/App.override.styles.scss';
+
+import './Badge';
+import './Button';
```

Comment:
Similarly, move this to `./core/Button` to follow its own path.

Reply:
same thing ...do you mind clarifying please? :)